### DISCUSSION TITLE
Howto instantly select a directory/file as I type?

### DISCUSSION THREAD

**Discussion by qrockz** [nvim-neo-tree/neo-tree.nvim#1185] [Q&A] [answered]
![tree](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/56853194/82b07695-6f4c-4b35-a6a3-870abfc3f12e)
For instance the current _selection_ is $**HOM**E but I want the selection jump to **Videos**.
How can I do that _without using ''j"_ to navigate there.

In the gui file browser Nemo i can start typing some letters and then the next matching file will get selected.

Or maybe show the line numbers before the tree entries so that I can jump to the specific line number.

**Comment by bew** [comment, marked as answer]
I personally disabled the default fuzzy search on `/` and would do `/vi` to make the cursor jump to the `Videos` item

**Reply by qrockz** [reply]
good idea!
